docs/richtext-commonmark.md

```diff
@@ -16,31 +16,40 @@ dependencies {
 The simplest way to render markdown is just pass a string to the [`Markdown`](/api/com.zachklipp.richtext.markdown/-markdown/)
 composable:
 
-```kotlin
+~~~kotlin
```

Comment:
Are these tildes to make nested fenced blocks work?

Reply:
exactly